Assets/Scripts/Runtime/Scriptable Objects/AutomaticSourceSpawning.cs

```diff
@@ -55,7 +57,12 @@ private Vector3Int GetSpawnPosition()
             var zCenter = (maxZ + minZ) / 2;
             var offset = new Vector3(xCenter, yCenter, zCenter);
 
-            var radius = 5f;
+            var averageDistToRelativeCenter = (
```

Comment:
maxX is not relative to the structure, so I think this calculation is incorrect. 

If maxX is 10 and minX is 0 then xCenter will be 5. And so the x-part of the formula would be equal to 10. 
Is this what we want? 
Would also appreciate having the x,y,z components of your calculation be in their own variables for clarity.

Reply:
I am deducting the center value, so in your example, x part of the formula would be 5, the max distance to the center, that was what I intended..

Reply:
Sorry I was confused by the stacked parenthesis.

Reply:
Can you break the calculation into multiple lines for readability?